config/contrib.json

```
@@ -6,7 +6,7 @@
     "versions": {
       "storage": {
         "git": {
-          "prefixMessageToSnapshotId": "This version was recorded after filtering snapshot with Mongo ID ",
+          "snapshotIdentiferTemplate": "open-terms-archive/snapshots/%SNAPSHOT_ID",
```

In my view we're losing too much information (Mongo and Contrib).
```suggestion
          "snapshotIdentiferTemplate": "mongo://contrib@open-terms-archive/snapshots/%SNAPSHOT_ID",
```